api/v1alpha1/zz_conversion.go

```diff
@@ -172,29 +167,29 @@ func fixupHostedControlPlaneAfterConversion(hcp *HostedControlPlane) error {
 	return populateDeprecatedGlobalConfig(hcp.Spec.Configuration)
 }
 
-func populatedDeprecatedAWSRoles(aws *AWSPlatformSpec) {
+func populateDeprecatedAWSRoles(aws *AWSPlatformSpec) {
 	aws.KubeCloudControllerCreds.Name = convertARNToSecretName(aws.RolesRef.KubeCloudControllerARN)
 	aws.NodePoolManagementCreds.Name = convertARNToSecretName(aws.RolesRef.NodePoolManagementARN)
 	aws.ControlPlaneOperatorCreds.Name = convertARNToSecretName(aws.RolesRef.ControlPlaneOperatorARN)
 	aws.Roles = []AWSRoleCredentials{
 		{
-			ARN:       aws.RolesRef.IngressARN,
-			Namespace: "openshift-ingress-operator",
+			ARN:       aws.RolesRef.NetworkARN,
```

Comment:
what is the purpose of reordering these?

Reply:
It was failing on comparison on unit tests. IIRC, the fuzzer was returning them in alphabetical order by namespace.